JavaImportTool/JavaImportTool.launch

```diff
@@ -8,20 +8,7 @@
 </listAttribute>
 <stringAttribute key="org.eclipse.debug.ui.ATTR_CAPTURE_IN_FILE" value="${workspace_loc:/JavaImportTool}/output.txt"/>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="javaImportTool.Main"/>
-
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="jedit /home/dasenov/Research/experiment1/jedit/source/ ${project_loc}/../DebugBuild/projects/"/>
-
-<!--
-===== The following are some useful ways to generate projects =====
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="legend /home/mitko/jedit/workspace/EnvisionLegend/src ${project_loc}/../DebugBuild/projects/"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="test ${project_loc}/test/brokencomments/ ${project_loc}/../DebugBuild/projects/"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="jedit /home/dasenov/Research/experiment1/jedit/source/ ${project_loc}/../DebugBuild/projects/"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="tetris ${project_loc}/test/JavaTetris/ ${project_loc}/../DebugBuild/projects/ -libs:java"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="large ${project_loc}/test/large_project/ ${project_loc}/../DebugBuild/projects/"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="sun ${project_loc}/test/java-sun/ ${project_loc}/../DebugBuild/projects/"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="java ${project_loc}/test/java-light/ ${project_loc}/../DebugBuild/projects/"/>
- -->
-
+<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="javaImportTool /home/guenatb/Java_Projects/javaImportTool /home/guenatb/Envision_Projects"/>
```

Comment:
Do not delete the old stuff, just add your line as a comment. Also note that this has to be changed in the place where your document flag. Please commit that file.